recall/ipld/src/hamt/map.rs

```diff
@@ -138,12 +138,12 @@ where
 
     pub fn get_or_create<F>(&self, key: &K, create_fn: F) -> Result<V, ActorError>
     where
-        F: FnOnce() -> V,
+        F: FnOnce() -> Result<V, ActorError>,
```

Comment:
I had to change this because [`Account::new`](https://github.com/recallnet/ipc/pull/541/files#diff-8f76f034a5bc328443f73e9c6bbb97e66dbaa18ef554803e85b4c4529b7d36a4R538-R539) returns `Result` now